### PR TITLE
vocabularies: add awards

### PR DESCRIPTION
**Pull request by alejandromumo**
This PR closes https://github.com/inveniosoftware/invenio-rdm-records/issues/918

### CODE REVIEW

File: tests/resources/vocabularies/test_awards_vocabulary.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    return current_service_registry.get("rdm-awards")
+
+
+@pytest.fixture()
+def example_award(
+    app, db, es_clear, superuser_identity, awards_service
+):
+    """Example award."""
+    data = {
+        "id": "755021",
```
Comment (alejandromumo) [inline tests/resources/vocabularies/test_awards_vocabulary.py:26]: Must be changed to `pid` when `PIDModelField` is implemented

File: invenio_rdm_records/ext.py [hunk 2]
```diff
@@ -18,6 +18,8 @@
 from invenio_vocabularies.contrib.affiliations import AffiliationsResource, \
     AffiliationsResourceConfig, AffiliationsService, \
     AffiliationsServiceConfig
+from invenio_vocabularies.contrib.awards import AwardsResource, \
```
Comment (ppanero) [inline invenio_rdm_records/ext.py:21]: update file copyright to 2022

Comment (ppanero) [inline invenio_rdm_records/ext.py:21]: update file copyright to 2022

**Review by ppanero — COMMENTED** [review]
All good! blocked till funders are merged.

File: invenio_rdm_records/views.py [hunk 3]
```diff
@@ -26,6 +26,7 @@ def init(state):
     registry.register(
         ext.records_service.draft_files, service_id='rdm-draft-files')
     registry.register(ext.affiliations_service, service_id='rdm-affiliations')
+    registry.register(ext.awards_service, service_id='rdm-awards')
```
Comment (ppanero) [inline invenio_rdm_records/views.py:29]:
```suggestion
    registry.register(ext.awards_service, service_id='awards-service')
```

File: tests/resources/vocabularies/test_awards_vocabulary.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                "scheme": "url"
+            }
+        ],
+        "number": "755021",
+        "title": {
+            "en": "Personalised Treatment For Cystic Fibrosis Patients With \
+                Ultra-rare CFTR Mutations (and beyond)",
+        },
+        "funder": {
+            # FIXME: add appropriate value when funders has been integrated
```
Comment (ppanero) [inline tests/resources/vocabularies/test_awards_vocabulary.py:39]: might be a good idea to tests also the dereferencing, i.e. save an award with only a `funder.id` and then query both the item endpoint (`/award/1231231`) and the search to test the dereferencing works at db and es level. Similar tests are done at invenio-vocabulares level (both in awards and names)

**Pull request closed without merging**